test/hummingbot/strategy/utils/test_historical_volatility.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from hummingbot.strategy.__utils__.trailing_indicators.historical_volatility import HistoricalVolatilityIndicator
+
+
+class HistoricalVolatilityIndicatorUnitTest(unittest.TestCase):
+
+    def test_historical_volatility(self):
```

Comment:
I think this unit test is not isolated enough. It appears to look more like an test for the entire class.
Unit tests should cover the smallest piece of code, i.e. `test_add_sample()`, `test_indicator_calculation()`, `test_processing_calculation()`.